screenpy/protocols.py

```diff
@@ -59,6 +62,22 @@ def perform_as(self, the_actor: "Actor") -> None:
         ...
 
 
+@runtime_checkable
+class ErrorKeeper(Protocol):
```

Comment:
I do think `ErrorKeeper` is the best name for it, even though `Exceptional` still has a place in my heart. <3